back/engines/free/email_campaigns/app/views/email_campaigns/idea_published_mailer/campaign_mail.mjml

```diff
@@ -74,7 +74,7 @@
         <td style="padding-left: 10px; padding-bottom:15px;">
           <%=
             format_message('action_share_twitter', values: {
-              twitterLink: "<a href=\"https://twitter.com/intent/tweet?text=#{CGI.escape event.post_url}\" style=\"text-decoration: underline;\">Twitter</a>"
+              twitterLink: "<a href=\"https://twitter.com/intent/tweet?text=#{CGI.escape event.post_url}\" style=\"text-decoration: underline;\">X</a>"
```

Comment:
@kogre Not sure if this was me or you, but if we change "Twitter" to "X", we probably also want to change the logo and URL. I think there was a task for this in "Things that annoy us"?

Reply:
Shall I revert to "Twitter" for now?